cpp/src/io/orc/dict_enc.cu

```diff
@@ -46,14 +47,16 @@ struct dictinit_state_s {
 };
 
 /**
- * @brief Return a 12-bit hash from a byte sequence
+ * @brief Return a 12-bit hash from a string
  */
-static inline __device__ uint32_t nvstr_init_hash(char const *ptr, uint32_t len)
+static inline __device__ uint32_t hash_string(const string_view &val)
```

Comment:
again, I think this should be passed by value, but libcudf is not very consistent. Will check.